packages/client/src/helpers/featureFlags.js

```diff
@@ -0,0 +1,3 @@
+export function useNewGrantsTable() {
+  return process.env.VUE_APP_USE_NEW_TABLE === 'true' || (window.APP_CONFIG || {}).UseNewTable === true;
+}
```

Comment:
@replicantSocks I left a [longer note in Slack](https://usdigitalresponse.slack.com/archives/C0324KDQSCR/p1690583565293829) about this, but if you decide to incorporate the [`feat/streamline-tf-website-feature-flags` branch](https://github.com/usdigitalresponse/usdr-gost/tree/feat/streamline-tf-website-feature-flags) into this PR, consider the following suggestion (or implementing something similar as you see fit):
```suggestion
export function useNewGrantsTable() {
  return process.env.VUE_APP_USE_NEW_TABLE === 'true' || ((window.APP_CONFIG || {}).featureFlags || {}).useNewTable === true;
}
```